src/background/clipboard.js

```diff
@@ -0,0 +1,21 @@
+const CLIPBOARD_CLEAR_DELAY = 1 * 60 * 1000; // 1 minute
+
+let clearClipboardTimer = null;
+
+export async function copyToClipboard(field, toCopy) {
+  if (clearClipboardTimer) {
+    // Clear any existing timer before scheduling a new one.
+    clearTimeout(clearClipboardTimer);
+  }
+  await navigator.clipboard.writeText(toCopy);
+  clearClipboardTimer = setTimeout(
+    async () => {
+      // Only clear the clipboard if it still holds the value set earlier.
```

Comment:
This bit wasn't specified in the issue, but it's a thing that's bitten me in other password managers: If I copy a password, then go on to use the clipboard for other things later - the manager clears unrelated data from my clipboard unexpectedly. Had to add a clipboard read permission to get it working, which may or may not be a problem.

Reply:
it wasn't called, it probably should have been called out, and it's great you called it out anyway (-: